lib/logger.js

```diff
@@ -10,34 +10,31 @@ exports.error = msg => console.error('[ERROR]', msg);
  * Error decoding/logging (some bigquery errors are very nested)
  */
 exports.errors = err => {
-  let errs = [];
-  if (err.errors) {
-    errs = errs.concat(err.errors.map(e => decodeErrors(e)));
+  if (err.response && err.response.insertErrors) {
+    err.message = exports.combineErrors(err);
   } else if (err.reason && err.message) {
-    errs.push(new Error(`${err.reason} - ${err.message}`));
-  } else {
-    errs.push(err);
-  }
-  errs.forEach(e => exports.error(`${e}`));
-
-  // return a single error
-  if (errs.length === 1) {
-    return errs[0];
-  } else if (errs.length > 1) {
-    let msgs = errs.map(e => `${e}`).join(', ');
-    return new Error(`Multiple errors: ${msgs}`);
-  } else {
-    return null;
+    err.message = `${err.reason} - ${err.message}`;
   }
+  exports.error(`${err}`);
+  return err;
 }
 
-// decode these crazy nested err.errors.errors
-function decodeErrors(err) {
```

Comment:
I'm not sure this was working correctly, since I did a major version upgrade on the node AWS SDK.  So I rewrote it.  Here's an example BigQuery SDK error, json'd:

```json
{
  "name": "PartialFailureError",
  "message": "A failure occurred during this request.",
  "errors": [
    {
      "errors": [
        {
          "message": "no such field.",
          "reason": "invalid"
        }
      ],
      "row": {
        "insertId": "708b01df9240db40bac587b82ed00d44",
        "json": {
          "timestamp": 1556725325,
          "user_id": "ccc3ec76551b99c14b42b689e735bcf9",
          "key": "test-runner-records1",
          "canonical": "https://www.prx.org/url1",
          "remote_agent": "some-user-agent",
          "remote_referrer": "https://www.prx.org/technology/",
          "remote_ip": "50.21.204.0",
          "city_geoname_id": 5576882,
          "country_geoname_id": 6252001,
          "postal_code": "80517",
          "latitude": 40.3658,
          "longitude": -105.5142
        }
      }
    },
    {
      "errors": [
        {
          "message": "no such field.",
          "reason": "invalid"
        }
      ],
      "row": {
        "insertId": "708b01df9240db40bac587b82ed00d44",
        "json": {
          "timestamp": 1556725325,
          "user_id": "ccc3ec76551b99c14b42b689e735bcf9",
          "key": "test-runner-records1",
          "canonical": "https://www.prx.org/url1",
          "remote_agent": "some-user-agent",
          "remote_referrer": "https://www.prx.org/technology/",
          "remote_ip": "50.21.204.0",
          "city_geoname_id": 5576882,
          "country_geoname_id": 6252001,
          "postal_code": "80517",
          "latitude": 40.3658,
          "longitude": -105.5142
        }
      }
    }
  ],
  "response": {
    "kind": "bigquery#tableDataInsertAllResponse",
    "insertErrors": [
      {
        "index": 0,
        "errors": [
          {
            "reason": "invalid",
            "location": "longitude",
            "debugInfo": "",
            "message": "no such field."
          }
        ]
      },
      {
        "index": 1,
        "errors": [
          {
            "reason": "invalid",
            "location": "longitude",
            "debugInfo": "",
            "message": "no such field."
          }
        ]
      }
    ]
  }
}
```